resources/js/components/fieldtypes/ProductQueryBuilder.vue

```diff
@@ -37,15 +62,36 @@ export default {
                 const response = await this.$axios.get('/cp/rapidez/product-attributes');
                 this.attributes = response.data.map(attr => ({
                     label: attr.store_frontend_label ? (attr.store_frontend_label + (` (${attr.attribute_code})`)) : attr.attribute_code,
-                    value: attr.attribute_code,
+                    value: `attribute.${attr.attribute_code}`,
                     type: this.mapAttributeType(attr.frontend_input),
                     options: this.mapAttributeOptions(attr.attribute_options)
                 }));
+
+                this.buildGroupedFields();
             } catch (error) {
                 console.error('Error fetching attributes:', error);
             }
         },
 
+        buildGroupedFields() {
+            this.groupedFields = this.fieldGroups.map(group => {
+                if (group.key === 'attribute') {
+                    return {
+                        label: group.label,
+                        options: this.attributes
+                    };
+                } else {
+                    return {
+                        label: group.label,
+                        options: group.fields.map(field => ({
+                            ...field,
+                            value: group.prefix ? `${group.prefix}${field.value}` : field.value
+                        }))
+                    };
+                }
+            });
```

Comment:
Don't think you really need the else here, this looks cleaner:
```suggestion
            this.groupedFields = this.fieldGroups.map(group => {
                if (group.key === 'attribute') {
                    return {
                        label: group.label,
                        options: this.attributes
                    };
                }
                
                return {
                    label: group.label,
                    options: group.fields.map(field => ({
                        ...field,
                        value: group.prefix ? `${group.prefix}${field.value}` : field.value
                    }))
                };
            });
```